docs/how-to-guides/running-air-gapped.md

```diff
@@ -0,0 +1,137 @@
+* Tar archives of Kubernetes k3s images. These are listed at https://github.com/k3s-io/k3s/releases, and you'll want to download `k3s-airgap-images-amd64.tar` or `k3s-airgap-images-arm64.tar` (for AMD/intel and M1 machines respectively) for the versions you plan on working with. For example, the following commands will let you work with K3s v 1.24.3 build 1 offline:
+
+```
+cd .../CACHE
+mkdir v1.24.3+k3s1
+cd v1.24.3+k3s1
+wget https://github.com/k3s-io/k3s/releases/download/v1.24.3%2Bk3s1/k3s-airgap-images-amd64.tar
+wget https://github.com/k3s-io/k3s/releases/download/v1.24.3%2Bk3s1/sha256sum-amd64.txt
+```
+
+##### Windows
+
+On Windows, the cache directory is at `%HOME%\AppData\Local\rancher-desktop\cache\k3s`, and can be created with the command
+
+```
+mkdir --Force %HOME%\AppData\Local\rancher-desktop\cache\k3s
+```
+
+Assuming you have some source media, you would also run the following commands to pre-populate the cache:
+
+```
+copy-item %SOURCEDISK%\k3s-versions.json %HOME%\AppData\Local\rancher-desktop\cache\
+copy-item -Recurse %SOURCEDISK%\v<MAJOR>.<MINOR>.<PATCH>+k3s<BUILD> %HOME%\AppData\Local\rancher-desktop\cache\k3s\
+```
+
+##### macOS
+
+On macOS, the cache directory is at `$HOME/Library/Caches/rancher-desktop` and the commands to populate it would be
+
+```
+CACHEDIR=$HOME/Library/Caches/rancher-desktop
+mkdir -p $CACHEDIR/k3s
+cp $SOURCEDISK/k3s-versions.json $CACHEDIR/
+cp -r $SOURCEDISK/v<MAJOR>.<MINOR>.<PATCH>+k3s<BUILD> $CACHEDIR/k3s/
+```
+
+##### Linux
+
+On Linux, the cache directory is at `$HOME/.cache/rancher-desktop` and the commands to populate it would be
+
+```
+CACHEDIR=$HOME/.cache/rancher-desktop
+mkdir -p $CACHEDIR/k3s
+cp $SOURCEDISK/k3s-versions.json $CACHEDIR/
+cp -r $SOURCEDISK/v<MAJOR>.<MINOR>.<PATCH>+k3s<BUILD> $CACHEDIR/k3s/
+```
+
+#### The kuberlr directory
+
+The location of this directory is more straightforward. On all platforms, it's at `HOME/.kuberlr/PLATFORM-ARCH` where `HOME` is the home directory on Windows (usually `%HOMEDRIVE%\%HOMEPATH` on Windows, and `~` or `$HOME` on macOS and Linux), `PLATFORM` is one of `windows`, `linux`, or `darwin`, and `ARCH` is `aarch64` on M1 machines, and `amd64` everywhere else.
+
+To populate it, determine which versions of Kubernetes you'll be using, and download the appropriate executables from the internet. These would be in:
+
+`https://dl.k8s.io/VERSION/bin/PLATFORM/CPU/kubectl`
+
+or for Windows:
+
+`https://dl.k8s.io/VERSION/bin/PLATFORM/CPU/kubectl.exe`
+
+`VERSION` will have the form `vMAJOR.MINOR.PATCH` (like `v1.22.1`),
+
+`PLATFORM` will be one of `darwin`, `linux`, or `windows`,
+
+`CPU` will `arm64` on M1 machines and `amd64` everywhere else.
```

Comment:
```suggestion
The location of this directory is more straightforward. On all platforms, it's at `HOME/.kuberlr/PLATFORM-ARCH`, where:

 - `HOME` is the home directory is usually `%HOMEDRIVE%\%HOMEPATH` on Windows, and `~` or `$HOME` on macOS and Linux.
 - `PLATFORM` is one of `windows`, `linux`, or `darwin`.
 - `ARCH` is `aarch64` on M1 machines, and `amd64` everywhere else.

To populate it, determine which versions of Kubernetes you'll be using, and download the appropriate executables from the internet. These would be in:

<Tabs groupId="os">
  <TabItem value="Windows">
  
`https://dl.k8s.io/VERSION/bin/PLATFORM/CPU/kubectl.exe`

  </TabItem>
  <TabItem value="macOS & Linux"
  
`https://dl.k8s.io/VERSION/bin/PLATFORM/CPU/kubectl`

  </TabItem>
</Tabs>

Where:

- `VERSION` will have the form `vMAJOR.MINOR.PATCH` (like `v1.22.1`),
- `PLATFORM` will be one of `darwin`, `linux`, or `windows`,
- `CPU` will `arm64` on M1 machines and `amd64` everywhere else.
```

Reply:
Better, but I made a slight change to the wording in the line describing `HOME`